charts/datasaur/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 2.1.11
```

Comment:
2.1.11 ini yang tadi merge ya? ditumpuk aja rob, we dont wanna create a "bugged" version @robbydrmwn

Reply:
updated dan @daniel-jeremia